documentation/en/payment-channels.md

```diff
@@ -0,0 +1,82 @@
+# Payment Channels
+
+Payment channels are used to transfer funds between two actors.
+
+In lotus a payment channel is created when a client wants to fetch data from a provider.
+The client sends vouchers for the payment channel, and the provider sends data in response.
+
+The payment channel is created on-chain with an initial amount.
+Vouchers allow the client and the provider to exchange funds incrementally off-chain.
+The provider can submit vouchers to chain at any stage.
+Either party to the payment channel can settle the payment channel on chain.
+After a settlement period (currently 12 hours) either party to the payment channel can call collect on chain.
+Collect sends the value of submitted vouchers to the channel recipient (the provider), and refunds the remaining channel balance to the channel creator (the client).
+
+Vouchers have a lane, a nonce and a value, where vouchers with a higher nonce supersede vouchers with a lower nonce in the same lane.
+Each deal is created on a different lane. 
+
+## Using the CLI
+
+For example a client creates a payment channel to a provider with value 10.
+
+```sh
+$ lotus paych get <client addr> <provider addr> 10
+<channel addr>
+```
+
+The client creates a voucher in lane 0 (implied) with nonce 1 (implied) and value 2.
+
+```sh
+$ lotus paych voucher create <channel addr> 2
+<voucher>
+```
+
+The client sends the voucher to the provider and the provider adds the voucher to their local store.
+
+```sh
+$ lotus paych voucher add <channel addr> <voucher>
+```
+
+The provider sends some data to the client.
+
+The client creates a voucher in lane 0 (implied) with nonce 2 (implied) and value 4.
+
+```sh
+$ lotus paych voucher create <channel addr> 4
+<voucher>
+```
+
+The client sends the voucher to the provider and the provider adds the voucher and sends back more data.
+etc.
+
+The client can add value to the channel after it has been created by calling `paych get` with the same client and provider addresses.
+
+```sh
+$ lotus paych get <client addr> <provider addr> 5
+<channel addr> # Same address as above. Channel now has 15
+```
+
+Once the client has received all their data, they settle the channel.
```

Comment:
I would make sure people know its not necessarily just for retrieval. Payment channels can be used for whatever you would like to pay people for

Reply:
Also note that settlement doesnt have to be done immediately, the same channel can be reused later for future exchanges